backend/internal/adapters/secondary/repositories/user.go

```diff
@@ -91,6 +94,31 @@ func (r *userRepository) Add(user *users.User) (int64, error) {
 		return 0, fmt.Errorf("unable to insert user: %w", err)
 	}
 
+	// Newly registered users are automatically friends with Gretel
+	var firstUserID, secondUserID int64
+	if constants.GretelUserId < userId {
+		firstUserID = constants.GretelUserId
+		secondUserID = userId
+	} else {
+		firstUserID = userId
+		secondUserID = constants.GretelUserId
+	}
+
+	sql, _, _ = qb.
+		Insert(goqu.T("friends")).
+		Rows(goqu.Record{
+			"first_user_id":  firstUserID,
+			"second_user_id": secondUserID,
+		}).
+		ToSQL()
+	tx.MustExec(sql)
```

Comment:
I learned from https://github.com/hansel-app/hansel/pull/147/commits/1cba3b6c6c363a40fd4d943d58a187adb41df1ba that this is quite bad since it crashes the entire backend.